erdblick_app/app/visualization.model.ts

```diff
@@ -20,44 +27,49 @@ interface StyleWithIsDeleted extends FeatureLayerStyle {
  * per map tile layer. Otherwise, they are rendered once per
  * (style sheet, tile layer) combination.
  */
-class LowDetailTileVisualization {
-    static visualizations: Map<string, LowDetailTileVisualization> = new Map<string, LowDetailTileVisualization>();
+class TileBoxVisualization {
+    static visualizations: Map<bigint, TileBoxVisualization> = new Map<bigint, TileBoxVisualization>();
 
-    static get(tile: FeatureTile, viewer: Viewer): LowDetailTileVisualization {
-        if (LowDetailTileVisualization.visualizations.has(tile.id)) {
-            let result = this.visualizations.get(tile.id)!;
+    static get(tile: FeatureTile, viewer: Viewer): TileBoxVisualization {
+        if (TileBoxVisualization.visualizations.has(tile.tileId)) {
+            let result = this.visualizations.get(tile.tileId)!;
             ++result.refCount;
             return result;
         }
 
-        return new LowDetailTileVisualization(viewer, tile);
+        return new TileBoxVisualization(viewer, tile);
     }
 
     // Keep track of how many TileVisualizations are using this low-detail one.
     // We can delete this instance, as soon as refCount is 0.
     refCount: number = 1;
     private readonly entity: Entity;
-    private readonly id: string;
+    private readonly id: bigint;
 
-    constructor(viewer: Viewer, tile: FeatureTile) {
-        let position = coreLib.getTilePosition(BigInt(tile.tileId));
-        let color = tile.numFeatures <= 0 ? Color.ALICEBLUE.withAlpha(.5) : Color.LAWNGREEN.withAlpha(.5);
+    constructor(viewer: Viewer,
+                tile: FeatureTile,
+                color: Color = Color.AQUA) {
+        let tileBox = coreLib.getTileBox(BigInt(tile.tileId));
+        // let color = tile.numFeatures <= 0 ? Color.ALICEBLUE.withAlpha(.5) : Color.LAWNGREEN.withAlpha(.5);
         this.entity = viewer.entities.add({
-            position: Cartesian3.fromDegrees(position.x, position.y),
-            point: {
-                pixelSize: 5,
-                color: color
+            rectangle: {
+                coordinates: Rectangle.fromDegrees(...tileBox),
+                height: HeightReference.CLAMP_TO_GROUND,
+                material: Color.TRANSPARENT,
+                outlineWidth: 2,
+                outline: true,
+                outlineColor: color

```

Comment:
The color should continue to reflect whether the tile is filled or not. From before:
```
let color = tile.numFeatures <= 0 ? Color.ALICEBLUE.withAlpha(.5) : Color.LAWNGREEN.withAlpha(.5);
```